libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -287,21 +324,22 @@ - (void)testCreateQuerySearchDelete {
     request = [[SFRestAPI sharedInstance] requestForQuery:[NSString stringWithFormat:@"select Id, FirstName from Contact where LastName='%@'", lastName]]; 
     listener = [self sendSyncRequest:request];
     XCTAssertEqualObjects(listener.returnStatus, kTestRequestStatusDidLoad, @"request failed");
-    NSArray *records = ((NSDictionary *)listener.dataResponse)[@"records"];
+    NSArray *records = ((NSDictionary *)listener.dataResponse)[RECORDS];
     XCTAssertEqual((int)[records count], 0, @"expected no result");
     
     // check the deleted object is here
     request = [[SFRestAPI sharedInstance] requestForQueryAll:[NSString stringWithFormat:@"select Id, FirstName from Contact where LastName='%@'", lastName]];
     listener = [self sendSyncRequest:request];
     XCTAssertEqualObjects(listener.returnStatus, kTestRequestStatusDidLoad, @"request failed");
-    NSArray* records2 = ((NSDictionary *)listener.dataResponse)[@"records"];
+    NSArray* records2 = ((NSDictionary *)listener.dataResponse)[RECORDS];
     XCTAssertEqual((int)[records2 count], 1, @"expected just one query result");
 
     // now search object
     request = [[SFRestAPI sharedInstance] requestForSearch:[NSString stringWithFormat:@"Find {%@}", lastName]];
     listener = [self sendSyncRequest:request];
     XCTAssertEqualObjects(listener.returnStatus, kTestRequestStatusDidLoad, @"request failed");
-    records = (NSArray *)listener.dataResponse;
+    records = ((NSDictionary *)listener.dataResponse)[SEARCH_RECORDS];
```

Comment:
Response body for sosl api call have changed recently from an array of results to a dictionary containing the results for the key searchRecords